src/portal-account.ts

```diff
@@ -0,0 +1,281 @@
+import jwt_decode from "jwt-decode";
+import { KeyPair, SkynetClient } from "skynet-js";
+import type { CustomClientOptions } from "skynet-js";
+import { sign } from "tweetnacl";
+
+import { genKeyPairFromHash, hashWithSalt } from "./crypto";
+import { hexToUint8Array, stringToUint8ArrayUtf8, toHexString, validateHexString, validateUint8ArrayLen } from "./util";
+
+// TODO: This is temporary for testing purposes. This should be changed to
+// `set-cookie` (will only work in the browser).
```

Comment:
F/U: Is this TODO still up to date? I thought we were unable to work with the `Set-Cookie` header from JS and `Skynet-Token` was to stay?